src/util/Constants.js

```diff
@@ -132,6 +132,10 @@ exports.Endpoints = {
         if (dynamic) format = hash.startsWith('a_') ? 'gif' : format;
         return makeImageUrl(`${root}/avatars/${userID}/${hash}`, { format, size });
       },
+      GuildMemberAvatar: (guildId, memberId, hash, format = 'webp', size, dynamic = false) => {
```

Comment:
Perhaps making these IDs like above for consistency sake?

Reply:
You mean using ID instead of Id? I believe it was like that when I made the PR or at least I saw that somewhere else but I can change it of course.